pyliquibase/__init__.py

```diff
@@ -93,7 +93,7 @@ def _cli(self):
         import jnius_config
 
         LIQUIBASE_CLASSPATH: list = [
-            self.liquibase_dir + "/liquibase.jar",
+            self.liquibase_dir,
```

Comment:
could you add `+ "/*"` to it, like the lines below? not sure this will work without `*`

Reply:
When I forked the project, I installed the library from the branch to test and it worked like this. But I can add it anyway, no worries